ZelBack/src/services/fluxCommunication.js

```diff
@@ -219,6 +219,57 @@ async function handleAppRunningMessage(message, fromIP, port) {
   }
 }
 
+/**
+ * To handle installing app messages.
+ * @param {object} message Message.
+ * @param {string} fromIP Sender's IP address.
+ * @param {string} port Sender's node Api port.
+ */
+async function handleAppInstallingMessage(message, fromIP, port) {
+  try {
+    // check if we have it exactly like that in database and if not, update
+    // if not in database, rebroadcast to all connections
+    // do furtherVerification of message
+    // eslint-disable-next-line global-require
+    const appsService = require('./appsService');
```

Comment:
I'm assuming this is done because of a dependency cycle - might be a bit of a perfomance impact importing this module on every installing message.

Can we move the `storeAppInstallingMessage` method to a common module, that both `fluxCommunication` and `appsService` import?

Seems like it would be a good fit on the `dbHelper` module?

Reply:
I disagree, I think it should be done on this PR - however, you're the boss.

The reasons I think it should be done on this PR:
* The function that created the dependency cycle was created in this PR
* The appsService module is enormous - dynamically importing that will have a large memory / time cost each time one of these messages is processed
* It will take 2 minutes to move the function to another module and do it properly

Agree, we should go and fix all the other requires that are like this (in another pull)